omod/src/main/java/org/openmrs/module/coreapps/fragment/controller/patientsearch/PatientSearchWidgetFragmentController.java

```diff
@@ -34,10 +34,18 @@ public void controller(FragmentModel model, UiSessionContext sessionContext,
                            @SpringBean("appFrameworkService") AppFrameworkService appFrameworkService,
                            @FragmentParam(value = "showLastViewedPatients", required = false) Boolean showLastViewedPatients,
                            @FragmentParam(value = "initialSearchFromParameter", required = false) String searchByParam,
-                           @FragmentParam(value = "registrationAppLink", required=false) String registrationAppLink) {
+                           @FragmentParam(value = "registrationAppLink", required=false) String registrationAppLink,
+                           @FragmentParam(value = "searchOnExactIdentifier", required=false) Boolean searchOnExactIdentifier) {
 
         showLastViewedPatients = showLastViewedPatients != null ? showLastViewedPatients : false;
 
+        // Determine version of OpenMRS to modify config parameter "searchOnExactIdentifier"
+        if(Double.valueOf(OpenmrsConstants.OPENMRS_VERSION_SHORT.substring(0,2)) >= 2.1) {
+            model.addAttribute("searchOnExactIdentifier", false);
+        }
+        else
+            model.addAttribute("searchOnExactIdentifier", true);
```

Comment:
By convention, we always used brackets around if/else, even if only a single line.